test/functionalTest/cases/1308_subscription_cache/cache_reset_from_db.test

```diff
@@ -0,0 +1,667 @@
+# Copyright 2015 Telefonica Investigacion y Desarrollo, S.A.U
```

Comment:
I understand that this new test in this merge was already reviewed in a previos PR in the 0.24.0 sub-branches. Is that correct? Just to save some reviewing time... ;)